config/config.go

```diff
@@ -19,13 +19,20 @@ type Config struct {
 	Ethereum ethereum.Config
 	LibP2P   libp2p.Config
 	Storage  Storage
+	Metrics  Metrics
 }
 
 // Storage stores meta-info about keeping data on disk
 type Storage struct {
 	DataDir string
 }
 
+// Metrics stores meta-info about metrics.
+type Metrics struct {
+	Port int
+	Tick int
```

Comment:
Can we have a separate tick for network and ETH metrics?

Reply:
Yep, sure.